homeassistant/components/velbus/cover.py

```diff
@@ -60,8 +62,9 @@ def current_cover_position(self) -> int | None:
         None is unknown, 0 is closed, 100 is fully open
         Velbus: 100 = closed, 0 = open
         """
-        pos = self._channel.get_position()
-        return 100 - pos
+        if pos := self._channel.get_position():
```

Comment:
Can `pos` be 0? Then we should probably check for `None` here explicitly.

Reply:
this can only an int, 0 to 100, will udpate the code